test/Models/CsvCatalogItemSpec.js

```diff
@@ -180,7 +484,8 @@ describe('CsvCatalogItem', function() {
             done();
         });
     });
-
+    /*
+    // Removed: not clear that this is correct behaviour, and it's failing.
```

Comment:
I don't know if transparent black is the right behavior.... but it's certainly possible to have a CSV file where the value column for most rows has a value, but it's missing for a few.  So what should happen in that case?  Transparent black (i.e. don't show the point or region) seems reasonable, but maybe there are other reasonable options.  In any case, being clear about what happens and writing a test for it seems worthwhile.